source/routing/raptor_api.cpp

```diff
@@ -125,6 +125,8 @@ call_raptor(navitia::PbCreator& pb_creator,
             };
             filter_late_journeys(raptor_journeys, params);
 
+            LOG4CPLUS_DEBUG(logger, "after filtering late journeys: " << raptor_journeys.size() << " solution(s) left");
```

Comment:
This is not the way I understood the need for log (though it's good to see how many are filtered in a single loop).
I though of a log describing the filtered journeys (summary).
Alternative being to tag them `to_delete` and `deleted_because_too_late_journey` when debug=True (but still send them to jormun, and then the user)...
The goal being to understand more easily why a certain journey doesn't appear in "normal" mode.